templates/paragraph/paragraph--section-fixed.html.twig

```diff
@@ -0,0 +1,99 @@
+ *   - paragraph.bundle(): The type of the paragraph, for example, "image" or "text".
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{% block attributes %}
+  {% set classes = [
+    'c-paragraph',
+    'c-paragraph--type-' ~ paragraph.bundle|clean_class,
+    view_mode ? 'c-paragraph--view-mode-' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'is-unpublished',
+  ] %}
+
+  {% set attributes = attributes.addClass(classes) %}
+{% endblock %}
+
+{% set paragraph_modifier_classes %}
+  {%- block modifier_classes -%}
+  {%- endblock -%}
+{% endset %}
+
+{% set paragraph_constrain_modifier_classes %}
+  {%- block constrain_modifier_classes -%}
+  {%- endblock -%}
+{% endset %}
+
+{% set paragraph_has_constrain %}
+  {%- block has_constrain -%}
+    {{ true }}
+  {%- endblock -%}
+{% endset %}
+
+{% set paragraph_title_element = 'h2' %}
+
+{% set num_of_cols = content.field_num_of_cols|render|striptags|trim %}
+
+{% set paragraph_content %}
+  {{ content['field_paragraphs_1'] }}
+  {{ content['field_paragraphs_2'] }}
+  {{ content['field_paragraphs_3'] }}
+{% endset %}
+
+{% embed '@layouts/section/section.twig' with {
+  'has_constrain': true,
+  'modifier_classes': paragraph_modifier_classes ~ ' l-section--two-one',
+  'constrain_modifier_classes': paragraph_constrain_modifier_classes,
+  'has_constrain': paragraph_has_constrain,
+  'section_kicker': content.field_text|field_value,
+  'section_title': content.field_short_text|field_value,
+  'section_title_element': paragraph_title_element,
```

Comment:
If this will always be `h2`, you can just write `section_title_element: 'h2'` rather than setting a variable for it up top.